shell/src/main/java/alluxio/shell/command/CpCommand.java

```diff
@@ -566,11 +567,12 @@ private void copyFileToLocal(AlluxioURI srcPath, AlluxioURI dstPath)
         out.write(buf, 0, t);
         t = is.read(buf);
       }
-      if (!tmpDst.renameTo(dstFile)) {
+      File outputFile = new File(dstFile.getAbsolutePath() + "/" + fileName));
```

Comment:
@maobaolong  the rename function can be true only if we set the destination filename, say rename (new File('A.txt'), new File('B.txt')). It will return false  if rename (new File('A.txt'), new File('/Users/Alluxio'))